react/features/breakout-rooms/actions.js

```diff
@@ -233,11 +233,12 @@ export function moveToRoom(roomId?: string) {
             const isVideoMuted = isLocalCameraTrackMuted(localTracks);
 
             try {
-                await APP.conference.leaveRoom(false /* doDisconnect */);
+                await APP.conference.leaveRoom(false /* doDisconnect */).then(
+                    () => APP.API.notifyConferenceLeft(APP.conference.roomName));
             } catch (error) {
                 logger.warn('APP.conference.leaveRoom() rejected with:', error);
 
-                // TODO: revisit why we don't dispatch CONFERENCE_LEFT here.
+                APP.API.notifyConferenceLeft(APP.conference.roomName);
```

Comment:
I think we should leave the comment because it is related to the CONFERENCE_LEFT action and not the IFrame API.

I'm also wondering if we are not dispatching the action should we trigger the confernce left IFrame API event? I mean we have failed to leave so we will probably still be in the room. WDYT?